revolv/static/js/base/dashboard.js

```diff
@@ -67,7 +67,7 @@ $(document).ready(function () {
      */
     function getCookie(name) {
         var cookieValue = null;
-        if (document.cookie && document.cookie != '') {
+        if (document.cookie && document.cookie !== '') {
```

Comment:
@sameerav you really need to do `v ssh -c "cd /vagrant && precommit-install"` lol